packages/runtime/src/prop-controllers/instances.ts

```diff
@@ -222,6 +227,9 @@ export function createPropController<T extends PropControllerMessage>(
     case RichTextControlType:
       return new RichTextControl(send as Send<RichTextControlMessage>)
 
+    case ListControlType:
+      return new ListControl(send as Send<unknown>, descriptor)
```

Comment:
Instead of `unknown`, shouldn't this be typed?

Reply:
Updated